src/main/java/io/cdap/e2e/utils/SeleniumHelper.java

```diff
@@ -77,6 +76,18 @@ public static void filemove(String name, String input) throws IOException, Inter
     }
   }
 
+  public static String readParameters(String property) throws IOException {
+    return connectProperties.getProperty(property);
+  }
+
+  public static <T> T getPropertiesLocators(Class<T> className) {
+    return PageFactory.initElements(SeleniumDriver.getDriver(), className);
+  }
+
+  /**
+   * @deprecated Use {@link ElementHelper#isElementDisplayed(WebElement)}
```

Comment:
Can we also call the new methods inside old deprecated methods to leverage the new changes??

Reply:
updated.